sdk/spring/spring-cloud-azure-integration-tests/pom.xml

```diff
@@ -97,11 +97,26 @@
       <artifactId>spring-cloud-azure-stream-binder-servicebus</artifactId>
       <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-stream-binder-servicebus;current} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-starter-jdbc-mysql</artifactId>
+      <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-stream-binder-servicebus;current} -->
+    </dependency>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-jdbc</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why are we using this?

Reply:
you mean h2?
I add spring-boot-starter-jdbc to test our spring-cloud-azure-starter-jdbc-mysql
so it will enable DataSourceAutoConfiguration
if url not provided, other ITs like keyvaultIT, cosmosIT will fail